setup.cfg

```diff
@@ -50,6 +50,7 @@ install_requires =
     pycryptodomex  # for EncryptedKeyring backend in keyrings.alt
     pydantic >= 1.9.0
     pynwb >= 1.0.3,!=1.1.0
+    nwbinspector @ git+https://github.com/NeurodataWithoutBorders/nwbinspector@validate_optional
```

Comment:
I'll update this once we have a new release with all the desired fixes in place for this PR